apps/prairielearn/src/pages/instructorAssessmentInstance/instructorAssessmentInstance.js

```diff
@@ -47,9 +47,24 @@ const DateDurationResultSchema = z.object({
 });
 
 const InstanceQuestionRowSchema = InstanceQuestionSchema.extend({
+  instructor_question_number: z.string().nullable(),
```

Comment:
Is this actually nullable?

Reply:
No. I now realize it is not nullable as the response is coming from the sproc `admin_assessment_question_number`, which always returns text. I have removed `.nullable()` in commit [7a88b09](https://github.com/PrairieLearn/PrairieLearn/pull/9250/commits/7a88b098dd62ef42b7196aa4cdafa015e6675a30).